modules/okhttp-client/src/main/java/com/spotify/apollo/http/client/HttpClient.java

```diff
@@ -38,11 +40,18 @@
 
   private final OkHttpClient client;
 
+  @Inject
   HttpClient(OkHttpClient client) {
     this.client = client;
   }
 
+  @Deprecated
+  /** @deprecated use {@link #createDefault()} instead. */
   public static HttpClient create() {
+    return createDefault();
+  }
+
+  public static HttpClient createDefault() {
```

Comment:
It didn't have to, but I wanted to make it clear that it was creating a HttpClient that was unconfigured / with all of the default configuration values